config/prod/minidream-R-env-test-instance-ec2.yaml

```diff
@@ -0,0 +1,20 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
```

Comment:
there is new version, please change to `templates/managed-ec2-v4.yaml`

Reply:
Updated versions.